### ISSUE TITLE
[Accessibility]: Enter web area shortcut on ref panel does nothing

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36575]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33510

### Problem description

On navigating via Tab, one reaches a "Tab panel" element (the focus looks like it is on the close button). The screen reader announces that you can try Control + Option + Shift + ↓ to enter the web content, but trying that shortcut doesn't do anything.

### Expected behavior

Either the shortcut shouldn't be announced, or there should be a change in focus on using that shortcut.

### Additional details

![](https://user-images.githubusercontent.com/93103176/171942965-9895a257-ae07-43b2-a755-a69ea850f320.png)

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
